pep-0011.txt

```diff
@@ -158,13 +158,15 @@ Microsoft has established a policy called product support lifecycle
 [1]_. Each product's lifecycle has a mainstream support phase, where
 the product is generally commercially available, and an extended
 support phase, where paid support is still available, and certain bug
-fixes are released (in particular security fixes).
+fixes are released. Sometimes this is then followed by a third phase
+called "ESU", for "extended security updates".
 
 CPython's Windows support now follows this lifecycle. A new feature
 release X.Y.0 will support all Windows releases whose extended support
-phase is not yet expired. Subsequent bug fix releases will support
-the same Windows releases as the original feature release (even if
-the extended support phase has ended).
+phase is not yet expired. (We don't consider the ESU phase for this
+purpose; only the "extended support" phase.) Subsequent bug fix releases
+will support the same Windows releases as the original feature release
+(even if the extended support phase has ended).
```

Comment:
```suggestion
phase is not yet expired. (ESU is not considered supported.)
Subsequent bug fix releases will support the same Windows releases
as the original feature release (even if the extended support phase has ended).
```

Following the above, clarifying that ESU is a specialized program, not a phase, and avoiding repeating ourselves about "extended support" too many times.